vars/snyk.groovy

```diff
@@ -27,10 +26,22 @@ def snykTest(Map snykTestArgs=[:]) {
             break;
     }
     withCredentials([string(credentialsId: "snyk-sa-token", variable: "TOKEN")]) {
-    sh """
-        snyk auth ${TOKEN}
-        ${xrayCommand}
-    """
+        sh """
+            snyk auth ${TOKEN}
+            ${xrayCommand}
```

Comment:
what does xrayCommand mean in this context?

Reply:
I had copied the variable name over from where I had done something similar in JFrog's file. Changing to snykTestCommand